src/pages/markets/static/content/_margin.js

```diff
@@ -12,14 +12,13 @@ import {
     StepIndices,
     VolatilityIndices,
 } from '../../instruments/_submarkets'
-import { smart_gold_index, step_fx } from '../../instruments/_market-symbols'
+import { smart_gold_index } from '../../instruments/_market-symbols'
 import {
     CrashBoomDetails,
     EuropeDetails,
     RangeBreakIndicesDetails,
     SmartFXDetails,
     SmartGoldIndexDetails,
-    StepFXDetails,
```

Comment:
I think you may have forgotten to remove `StepFXDetails` from `markets/static/content/_details.js` as well.

Reply:
That has already been removed; it is already highlighted in red bg.

Reply:
Nono, I am referring to the one in the `_details.js` file.

Reply:
Nono, I am referring to the one in the `_details.js` file.

Reply:
Gotcha! on it!

Reply:
Updated on this PR, kindly check it out, thanks.... https://github.com/binary-com/deriv-com/pull/1567/files